src/apps/frontend/pages/header/header.tsx

```diff
@@ -0,0 +1,39 @@
+import { HeaderNavigation } from 'baseui/header-navigation';
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { useAuthContext } from '../../contexts';
+
+import NavbarDesktopCenterSection from './navbar-sections/navbar-desktop-center-section';
+import NavbarDesktopLeftSection from './navbar-sections/navbar-desktop-left-section';
+import NavbarDesktopRightSection from './navbar-sections/navbar-desktop-right-section';
+
+interface HeaderNavbarProps {}
+
+const Header: React.FC<HeaderNavbarProps> = () => {
+  const navigate = useNavigate();
+  const { signOut } = useAuthContext();
+
+  const handleSignOut = () => {
+    signOut();
+    navigate(0);
+  };
+
+  const profileMenuItems = [
+    {
+      label: 'Logout',
+      onClickHandler: handleSignOut,
+    },
+  ];
+  return (
+    <HeaderNavigation>
+      <NavbarDesktopLeftSection />
+      <NavbarDesktopCenterSection />
+      <NavbarDesktopRightSection
```

Comment:
Why `Desktop`?

In a responsive web, I don't expect to see separate `web` or `mobile` components. The rendering / stacking of components can change but no specific components for web or mobile

Reply:
Addressed